### PR TITLE
Change from 'is' to 'are'

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Change from 'is' to 'are' with note messaging.

### CODE REVIEW

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/standard/serialization/system-text-json-character-casing.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ In this article, you will learn how to enable case-insensitive property name mat
 By default, deserialization looks for case-sensitive property name matches between JSON and the target object properties. To change that behavior, set <xref:System.Text.Json.JsonSerializerOptions.PropertyNameCaseInsensitive?displayProperty=nameWithType> to `true`:
 
 > [!NOTE]
-> The [web defaults](system-text-json-configure-options.md#web-defaults-for-jsonserializeroptions) is case-insensitive.
+> The [web defaults](system-text-json-configure-options.md#web-defaults-for-jsonserializeroptions) are case-insensitive.
```
Comment (tdykstra) [inline docs/standard/serialization/system-text-json-character-casing.md:22]:
```suggestion
> The [web default](system-text-json-configure-options.md#web-defaults-for-jsonserializeroptions) is case-insensitive.
```

**Review by tdykstra — APPROVED** [review]
On second thought maybe "default is" would be better than "defaults are"

File: docs/standard/serialization/system-text-json-customize-properties.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ helpviewer_keywords:
 By default, property names and dictionary keys are unchanged in the JSON output, including case. Enum values are represented as numbers. In this article, you'll learn how to:
 
 > [!NOTE]
-> The [web defaults](system-text-json-configure-options.md#web-defaults-for-jsonserializeroptions) is camel case.
+> The [web defaults](system-text-json-configure-options.md#web-defaults-for-jsonserializeroptions) are camel case.
```
Comment (IEvangelist) [inline docs/standard/serialization/system-text-json-customize-properties.md:18]:
```suggestion
> The [web default](system-text-json-configure-options.md#web-defaults-for-jsonserializeroptions) is camel case.
```

**Pull request merged**